plugins/Jobs.cpp

```diff
@@ -265,7 +265,8 @@ bool BedrockPlugin_Jobs::processCommand(BedrockNode* node, SQLite& db, BedrockNo
             SQResult result;
             if (!db.read("SELECT 1 "
                          "FROM jobs "
-                         "WHERE jobID = " + safeParentJobID + ";",
+                         "WHERE jobID = " +
+                             safeParentJobID + ";",
```

Comment:
Ooh, you've merged this with the priority stuff too?  I guess this is safe to merge?